environ/environ.py

```diff
@@ -191,6 +190,8 @@ class Env:
         "xapian": "haystack.backends.xapian_backend.XapianEngine",
         "simple": "haystack.backends.simple_backend.SimpleEngine",
     }
+    ELASTICSEARCH_FAMILY = [scheme + s for scheme in SEARCH_SCHEMES if scheme.startswith("elasticsearch")
```

Comment:
line too long (105 > 79 characters)

Reply:
Thanks for reviewing @sergeyklay.
I've hit the statement limit for this method, though. What do you want me to do about it?
```  lint: commands[1]> pylint --logging-format-style=old '--good-names-rgxs=m[0-9],f,v' --disable=too-few-public-methods --disable=import-error --disable=unused-import --disable=too-many-locals --disable=too-many-branches --disable=too-many-public-methods --disable=too-many-lines environ
  ************* Module environ.environ
  environ/environ.py:746:4: R0915: Too many statements (55/50) (too-many-statements)```
Add `--disable=too-many-statements`? Splitting the method would seem out of character for this module.
And I couldn't optimize the number of statements in this method without touching older code and/or losing clarity.